### PR TITLE
add r9slim (old r9slim with 1 antenna port) target

### PR DESCRIPTION
**Pull request by StonedDawg**
PeHo and JesusGonfer has been helping to add r9slim RX as elrs RX, for now only stlink method can be used.

### CODE REVIEW

File: src/platformio.ini [hunk 1]
```diff
@@ -140,7 +140,48 @@ upload_flags =
...
+build_unflags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_unflags}
+build_flags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_flags}
+board_build.ldscript = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.board_build.ldscript}
+src_filter = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.src_filter}
+upload_flags =
+    BOOTLOADER=bootloader/r9slim_no_btn_bootloader.bin
+    VECT_OFFSET=0x8000
+lib_deps = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.lib_deps}
+
+[env:Frsky_RX_R9SLIM_dump_via_STLINK]
```
Comment (cruwaller) [inline src/platformio.ini:155]: remove this target.

File: src/platformio.ini [hunk 2]
```diff
@@ -140,7 +140,48 @@ upload_flags =
...
+build_unflags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_unflags}
+build_flags = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.build_flags}
+board_build.ldscript = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.board_build.ldscript}
+src_filter = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.src_filter}
+upload_flags =
+    BOOTLOADER=bootloader/r9slim_no_btn_dump_bootloader.bin
+    VECT_OFFSET=0x8000
+lib_deps = ${env:Frsky_RX_R9SLIM_via_BetaflightPassthrough.lib_deps}
 
+[env:Frsky_RX_R9SLIM_via_BetaflightPassthrough]
```
Comment (cruwaller) [inline src/platformio.ini:167]: Check `extends = `

File: src/platformio.ini [hunk 3]
```diff
@@ -140,7 +140,48 @@ upload_flags =
 src_filter = ${env_common_stm32.src_filter} -<rx_*.cpp>
 lib_deps =
 	https://github.com/PaoloP74/extEEPROM.git
+[env:Frsky_RX_R9SLIM_via_STLINK]
```
Comment (cruwaller) [inline src/platformio.ini:143]: Check `extends = ` usage.

File: src/bootloader/src/platformio.ini [hunk 4]
```diff
@@ -62,7 +62,89 @@ build_flags =
...
+#board_build.mcu = ${env:R9SLIM.board_build.mcu}
+#board_build.f_cpu = ${env:R9SLIM.board_build.f_cpu}
+board_build.ldscript = ${env:R9SLIM.board_build.ldscript}
+board_upload.maximum_size = 16384
+build_flags =
+    ${r9slim_generic.flags}
+    -Wl,--defsym=FLASH_OFFSET=0x2000
+    -D FLASH_APP_OFFSET=0x8000u
+
+[env:R9SLIM_no_btn_dump]
```
Comment (cruwaller) [inline src/bootloader/src/platformio.ini:122]: remove this target

File: src/bootloader/src/platformio.ini [hunk 5]
```diff
@@ -62,7 +62,89 @@ build_flags =
...
+#board_build.f_cpu = ${env:R9SLIM.board_build.f_cpu}
+board_build.ldscript = ${env:R9SLIM.board_build.ldscript}
+board_upload.maximum_size = 16384
+build_flags =
+    ${r9slim_generic.flags}
+    -Wl,--defsym=FLASH_OFFSET=0x0
+    -D FLASH_APP_OFFSET=0x8000u
+    -D DUMPING=1
+
+[env:R9SLIM_no_btn_stock_dump]
```
Comment (cruwaller) [inline src/bootloader/src/platformio.ini:134]: remove this target

**Review by cruwaller — CHANGES_REQUESTED** [review]

File: src/platformio.ini (hunk 1 above)
Comment (StonedDawg) [reply src/platformio.ini]: sorry, I thought I removed them but turns out I didn't save before push

**Review by cruwaller — APPROVED** [review]

**Pull request merged**